tracim/tracim/lib/email.py

```diff
@@ -73,11 +73,10 @@ def connect(self):
                 try:
                     starttls_result = self._smtp_connection.starttls()
                     logger.debug(self, 'SMTP start TLS result: {}'.format(starttls_result))
+                    login_res = self._smtp_connection.login(self._smtp_config.login, self._smtp_config.password)
```

Comment:
Your fix is not working in case we do not want TLS but want login (starttls will fail, then login will never be checked).

Could you put this in a if statement after the try/except so that the login is not executed if empty
something like that:
```
if self._smtp_config.login:
    login_res = self._smtp_connection.login(self._smtp_config.login, self._smtp_config.password)
    logger.debug(self, 'SMTP login result: {}'.format(login_res))
```

Reply:
i pushed the changes.

From my point of view there is a remaining problem : 
It can log at debug level the starttls() or login() exceptions, but this is not handled at the next logger.info() call which log "Connection OK" whatever happens.
I don't know what's your policy for exceptions handling/logging in this kind of case, but it can be pretty disturbing if you don't have debug enabled or if you think that the debug info which is logged is just an info but everything went fine...as it's not true.

In this example my login/password pair was not valid, and the mail was never sent :
```
22:15:01,501 INFO  [tracim] [EmailSender] Connecting from SMTP server srv06.gecoh-regence.local
22:15:06,510 INFO  [tracim] [EmailSender] Connection OK
22:15:06,511 INFO  [tracim] [EmailSender] Sending email to xxx@xxx.xxx
```